assets/js/player.js

```diff
@@ -112,6 +112,37 @@ function addCurrentTimeToURL(url) {
     return urlUsed;
 }
 
+/**
+ * Callback that updates the timestamp on all external links
+ */
+player.on('timeupdate', function () {
+    // Only update once every 5 seconds
+    if ((Math.ceil(player.currentTime()) % 5) != 0) return;
+
+    // YouTube links
+
+    let elem_yt_watch = document.getElementById('link-yt-watch');
+    let elem_yt_embed = document.getElementById('link-yt-embed');
```

Comment:
Actually now that I think about it does it even make sense to include the timestamp in the embed link? YouTube doesn't support the `t=` url parameter and instead wants you to use their iframe API. 

Invidious currently does support the `t=` parameter on embeded videos but once #4060 gets merged maybe we should have the same behavior?

Reply:
As stated in #3348, it's hard to manually add the timestamp to an URL. But if the timestamp is present, users can easily remove it.

As for iframes, I'm not very found of forcing the use of an API. Sure, this can be another way of doing it, but why not supporting both?

Reply:
> ... #3348

I believe that issue is more about adding a separate "Watch on YouTube" link with a timestamp on it to the embedded player.
 
> but why not supporting both?

Fair enough. I was mostly thinking of a weird edge case where the `t=` parameter could impact compatibility between the YouTube and Invidious embed players but that's probably something we don't have to worry about.

Reply:
> I believe that issue is more about adding a separate "Watch on YouTube" link with a timestamp on it to the embedded player.

Oooh, you're right! I misunderstood the issue!
We probably need a much better menu for videoJS though :/